homeassistant/components/fritzbox/sensor.py

```diff
@@ -131,7 +131,7 @@ class FritzSensorEntityDescription(
         device_class=DEVICE_CLASS_TIMESTAMP,
         suitable=lambda device: device.has_thermostat
         and device.nextchange_endperiod is not None,
-        native_value=lambda device: datetime.fromtimestamp(
+        native_value=lambda device: dt.utc_from_timestamp(
             device.nextchange_endperiod
         ).isoformat(),
```

Comment:
`isoformat()` is not needed when using `utc_from_timestamp()`

Reply:
using `utc_from_timestamp()` would return a `datetime` which is not allowed in `StateType` at the moment.
See https://github.com/home-assistant/core/pull/52671

will go with string instead in the meanwhile like other integrations do as well at the moment

Reply:
ah ok ... thanks for pointing this out - was not aware of this ongoing discussion 👍